ide/projectapi.nb/manifest.mf

```diff
@@ -1,6 +1,6 @@
 Manifest-Version: 1.0
 OpenIDE-Module: org.netbeans.modules.projectapi.nb
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/projectapi/nb/Bundle.properties
-OpenIDE-Module-Provides: cnb.org.netbeans.modules.projectapi.nb
-OpenIDE-Module-Specification-Version: 1.13
+OpenIDE-Module-Provides: org.netbeans.spi.project.ProjectManagerImplementation
```

Comment:
@tzezula, there is no need to `OpenIDE-Module-Provides: cnb.org.netbeans.modules.projectapi.nb` it is provided automatically. Rather than that the module no provides implementation of `org.netbeans.spi.project.ProjectManagerImplementation`.